jobs/seasonal/stats/JEVS_SEASONAL_STATS

```diff
@@ -1 +1,115 @@
-sample
+#!/bin/sh
+
+set -xa
+
+# ###################################
+# SET SHELL PROCESSING VARIABLES
+# ###################################
+export PS4='$SECONDS + ' 
+date
+
+export machine="WCOSS2"
+
+###########################################################
+# obtain unique process id (pid) and make temp directories
+###########################################################
+export pid=$$
+#export DATA_IN=${DATA_IN:-$DATAROOT}
+export DATA_IN=${DATA_IN:-$DATAROOTtmp}
+export DATA=$DATA_IN/${jobid}
+
+
+rm -rf $DATA
+mkdir -p $DATA
+cd $DATA 
+
+###############################
+# Determine Job Output Name on System
+###############################
+export pgmout="OUTPUT.${pid}"
+
+#export cycle=t${cyc}z 
+
+
+################################################################
+# SENDCOM=YES--Copy files from TMPDIR to $COMOUT
+# SENDECF=YES--Flag events on ecflow
+# SENDDBN=YES--Issue DBNet client calls
+################################################################
+export SENDCOM=${SENDCOM:-NO}
+export SENDDBN=${SENDDBN:-NO}       # need to set to NO for testing
+export SENDECF=${SENDECF:-NO}
+export SENDDBN_NTC=${SENDDBN_NTC:-NO}
+################################################################
+# Specify Execution Areas
+################################################################
+export DCOMROOT=${DCOMROOT:-/lfs/h1/ops/$envir/dcom}
+#export HOMEevs=${HOMEevs:-${PACKAGEROOT}/$NET/$evs_ver}
+export EXECevs=${EXECevs:-$HOMEevs/exec}
+export PARMevs=${PARMevs:-$HOMEevs/parm}
+export USHevs=${USHevs:-$HOMEevs/ush}
+export FIXevs=${FIXevs:-$HOMEevs/fix}
+#export HOMEMET=${HOMEMET}
+#export HOMEMETplus=${HOMEMETplus}
+#export PARMMETplus=${PARMMETplus:-$HOMEMETplus/parm}
+#export USHMETplus=${USHMETplus:-$HOMEMETplus/ush}
+#export PATH="${USHMETplus}:${PATH}"
+#export PYTHONPATH="${USHMETplus}:${PYTHONPATH}"
+
+export members="4"
+
+##################################
+# Set up the UTILITIES
+###################################
+# for wwgrib: find the path of wgrib2: 
+#export WGRIBpath=`echo $WGRIB | sed 's/\/wgrib//'`
+
+
+# Run setpdy and initialize PDY variables
+#setpdy.sh
+#$. ./PDY
+
+#$export vday=$PDYm1
+export VDATE=$(date -d "today -1 day" +"%Y%m%d")
+
```

Comment:
Let's change this VDATE section. The VDATE definition in the j-job should be able to inherit a value coming from the ecf script if it has been set. If VDATE has not been set in the ecf script, then the default value in the j-job will be used.
Change line 72 to:
`export VDATE=${VDATE:-$PDYm1}`
Remove line 73. The override value for VDATE will be set in the ecf script.